packages/fast-usdc/tools/mock-evidence.ts

```diff
@@ -151,6 +153,50 @@ export const MockCctpTxEvidences: Record<
     },
     chainId: 8453,
   }),
+  AGORIC_PLUS_NOBLE: (receiverAddress?: Bech32Address) => ({
+    blockHash:
+      '0x80d7343e04f8160892e94f02d6a9b9f255663ed0ac34caca98544c8143fee6z9',
+    blockNumber: 21037600n,
+    blockTimestamp,
+    txHash:
+      '0xe81bc6105b60a234c7c50ac17816ebcd5561d366df8bf3be59ff3875527617z9',
+    tx: {
+      amount: 950000000n,
+      forwardingAddress: 'noble17ww3rfusv895d92c0ncgj0fl9trntn70jz7ee5',
+      sender: Senders.default,
+    },
+    aux: {
+      forwardingChannel: 'channel-21',
+      recipientAddress:
+        receiverAddress ||
+        encodeAddressHook(settlementAddress.value, {
+          EUD: 'cosmos:noble-1:noble1u2l9za2wa7wvffhtekgyuvyvum06lwhqxfyr5d',
+        }),
+    },
+    chainId: 8453,
+  }),
+  AGORIC_PLUS_NOBLE_BECH32: (receiverAddress?: Bech32Address) => ({
```

Comment:
this name threw me off. please include a JSDoc for how it differs from `AGORIC_PLUS_NOBLE`. Also consider a name like `AGORIC_PLUS_NOBLE_B32EUD`

Reply:
done